gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/EncassLinker.java

```diff
@@ -27,6 +32,16 @@ public void link(Encass encass, Bundle bundle, Bundle targetBundle) {
         if (policy == null) {
             throw new LinkerException("Could not find policy for Encapsulated Assertion: " + encass.getName() + ". Policy ID: " + encass.getPolicyId());
         }
+        Element encassPortalIntegrationElement = null;
+        try {
+            encassPortalIntegrationElement = getSingleElement(policy.getPolicyDocument(), API_PORTAL_ENCASS_INTEGRATION);
+        } catch (DocumentParseException ex) {
+            // do nothing
+        }
+        if (encassPortalIntegrationElement != null) {
+            policy.getPolicyDocument().getFirstChild().removeChild(encassPortalIntegrationElement);
+        }
+        encass.getProperties().put(PORTAL_TEMPLATE, encassPortalIntegrationElement != null ? "true" : "false");
```

Comment:
Why string? you can directly put boolean.

Reply:
@nikhilahuja , this is to inline with existing boolean properties which are represented as strings in encass YML file